pandas_gbq/features.py

```diff
@@ -0,0 +1,95 @@
+"""Module for checking dependency versions and supported features."""
```

Comment:
I pulled this out of `gbq.py`, because `load.py` also needs some of this logic now.